rasa/core/channels/socketio.py

```diff
@@ -98,6 +98,12 @@ def __init__(self, sio, sid, bot_message_evt):
 
         await self.sio.emit(self.bot_message_evt, **json_message)
 
+    async def send_attachment(
+        self, recipient_id: Text, attachment: Iterable[Dict[Text, Any]], **kwargs: Any
```

Comment:
Why `Iterable`? Should be a `Dict`, right?
```suggestion
        self, recipient_id: Text, attachment:Dict[Text, Any], **kwargs: Any
```